pennylane/io.py

```diff
@@ -122,11 +109,177 @@ def from_qiskit(quantum_circuit, measurements=None):
+        >>> circuit()
+        [tensor(-1., requires_grad=True), tensor(1., requires_grad=True)]
+
+    See below for more information regarding how to translate more complex circuits
+    from Qiskit to PennyLane, including handling parameterized Qiskit circuits,
+    mid-circuit measurements, and classical control flows.
+
+    .. details::
+        :title: Parameterized Quantum Circuits
+
+        A Qiskit ``QuantumCircuit`` is parameterized if it contains ``Parameter`` or
+        ``ParameterVector`` references that need to be given defined values to evaluate
+        the circuit. These can be passed to the generated qfunc as keyword or positional
+        arguments. If we define a parameterized
+
+        .. code-block:: python
+
+            from qiskit.circuit import QuantumCircuit, Parameter
+
+            angle0 = Parameter("x")
+            angle1 = Parameter("y")
+
+            qc = QuantumCircuit(2, 2)
+            qc.rx(angle0, 0)
+            qc.ry(angle1, 1)
+            qc.cx(1, 0)
+
+        Then this circuit can be converted into a differentiable circuit in PennyLane and
+        executed:
+
+        .. code-block:: python
+
+            import pennylane as qml
+            from pennylane import numpy as np
+
+            dev = qml.device("default.qubit")
+
+            @qml.qnode(dev)
+            def circuit(a0, a1):
+                # parameters are passed from the circuit function into the template as keyword arguments
+                # the keywords match the names given to the parameters in the code block above
+                qml.from_qiskit(qc)(x=a0, y=a1)
+                return qml.expval(qml.Z(0))
+
+        .. note::
+
+            The parameters can also be passed to the qfunc as positional arguments. In this case, the positions
+            match those from ``qc.parameters``. Note that these are in alphabetical order by name, rather than
+            in the order they are used in the circuit.
+
+        >>> circuit(np.pi/4, 0.)
+        tensor(0.70710678, requires_grad=True)
+
+        >>> qml.grad(circuit, argnum=[0, 1])(np.pi/4, np.pi/6)
+        (array(-0.61237244), array(-0.35355339))
+
+        The ``QuantumCircuit`` may also be parameterized with a ``ParameterVector``. These can be similarly
+        converted:
+
+        .. code-block:: python
+
+            from qiskit.circuit import ParameterVector
+
+            angles = ParameterVector("angles", 2)
+
+            qc = QuantumCircuit(2, 2)
+            qc.rx(angles[0], 0)
+            qc.ry(angles[1], 1)
+            qc.cx(1, 0)
+
+            @qml.qnode(dev)
+            def circuit(angles):
+                qml.from_qiskit(qc)(angles)
+                return qml.expval(qml.Z(0))
+
+        >>> angles = [3.1, 0.45]
+        >>> circuit(angles)
+        tensor(-0.89966835, requires_grad=True)
+
+        #TODO: either note here that this doesn't work with gradients, or get the gradient working and add qml.grad(circuit, argnum=0)([np.pi/4, np.pi/6])
```

Comment:
```suggestion
```
We probably don't need to mention the subtlety of param-vectors also being differentiable.

Reply:
@lillian542 Any objections to this removal?